src/popup/router/pages/Networks.vue

```diff
@@ -201,4 +219,30 @@ export default {
   margin-left: 5px;
   margin-right: 0;
 }
+
+.expand {
+  font-size: 14px;
+  cursor: pointer;
+
+  span {
+    display: inline-block;
+  }
+
+  img {
+    transform: rotate(-90deg);
+    vertical-align: middle;
+
+    & ~ span {
+      margin-bottom: 20px;
```

Comment:
Instead, you can apply this style to the buttons below to don't switch them on opening